test/cmdlineTests/strict_asm_optimizer_steps/output

```diff
@@ -16,6 +16,7 @@ object "C_6" {
             {
                 mstore(64, 128)
                 pop(iszero(lt(calldatasize(), 4)))
+                mstore(96, iszero(calldatasize()))
```

Comment:
I think it would be better to use `sstore`, we might optimizer `mstore` away in the future.

Reply:
Replaced with `sstore`.